src/webgpu/api/validation/createTexture.spec.ts

```diff
@@ -4,11 +4,6 @@ createTexture validation tests.
 TODO: review existing tests and merge with this plan:
```

Comment:
I noticed not a lot of the old tests got replaced in your past few PRs on this file. It's okay to have duplicates but could you confirm there are at least no _obvious_ candidates to deduplicate?

Reply:
The old tests are basically removed (or replaced?) because they are too simple. Currently I think there are no significant duplicates. You see, every test focus on one single parameter or a small field for all formats, like mipLevelCount, sampleCount, maxTextureSize, texture-usage, zero-size-not-allowed, etc. There might be small overlap but I am trying to avoid: for example, for the conflict between multisample support and STORAGE usage, It can be tested in sampleCount or texture usage. So I only test it in sampleCount related test, but don't test it again at texture usage.

Reply:
Thanks for checking this!